### ISSUE TITLE
Delete all of the out-of-date mumbo jumbo from the "im" howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4054]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4054|
|Type|defect|
|Created|2009-10-10 02:59:21Z|
|Branch|https://github.com/twisted/twisted/tree/im-doc-4054|

Twisted Words has little documentation.  It only has one howto, in fact.  And it's a pretty bad one.

Offhand, here's some stuff it says that is wrong:

  * The whole "Note" at the beginning is wrong.  Twisted Words isn't in a state of flux and hasn't been at any point since that note was added.  Delete it.
  * There's no such thing as `twisted/scripts/im.py` anymore.  In fact, there doesn't seem to be _any_ way to start im anymore.
  * There is no longer a swing front end.
  * The doc talks about `twisted.im` which is gone now.

There's probably plenty of other stuff wrong with the doc, but these are the most obvious falsehoods which can be corrected simply by deletion.

<details><summary>Searchable metadata</summary>

```
trac-id__4054 4054
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_im_doc_4054 branches/im-doc-4054
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__words words
keywords__documentation__easy documentation, easy
time__1255143561000000 1255143561000000
changetime__1295213963000000 1295213963000000
version__None None
owner__ 
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [30461]) Branching to 'im-doc-4054'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [30462]) Remove outdated information from the howto.

refs [#4054](https://github.com/twisted/twisted/issues/4054)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Replying to [exarkun](#4054):
>   * The whole "Note" at the beginning is wrong.  Twisted Words isn't in a state of flux and hasn't been at any point since that note was added.  Delete it.

Done.

>   * There's no such thing as `twisted/scripts/im.py` anymore.  In fact, there doesn't seem to be _any_ way to start im anymore.
>   * There is no longer a swing front end.

Removed the first section of the 'Code Flow' paragraph.

>   * The doc talks about `twisted.im` which is gone now.

Replaced with references to `twisted.words.im`.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Thanks.  I think the news file could be a little more precise, like "Outdated GUI-related information was removed from the IM howto." (note also the change in tense).

Otherwise looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [30466]) Provide better description for doc change.

refs [#4054](https://github.com/twisted/twisted/issues/4054)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [30468]) Merge im-doc-4054: Outdated GUI-related information was removed from the IM howto.

Author: thijs
Reviewer: exarkun
Fixes: [#4054](https://github.com/twisted/twisted/issues/4054)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
